recipe/meta.yaml

```
@@ -37,16 +36,16 @@ requirements:
     - fftw 3.3.9
     - glpk 4.65
     - gsl 2.7.1
-    - openblas-devel {{ openblas }}  # [blas_impl == "openblas"]
+    - openblas-devel {{ openblas_devel }}  # [blas_impl == "openblas"]
     # suitesparse is linked against on windows, and compiled-in on windows, because yeah, windows.
-    - suitesparse 5.10.1            # [not win]
+    - suitesparse 7.8.3  # [not win]
```

```suggestion
    - suitesparse {{ suitesparse }}  # [not win]
```
https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L752